src/Symfony/Component/RateLimiter/Policy/SlidingWindow.php

```diff
@@ -84,9 +84,34 @@ public function getHitCount(): int
         return (int) floor($this->hitCountForLastWindow * (1 - $percentOfCurrentTimeFrame) + $this->hitCount);
     }
 
+    /**
+     * @deprecated since Symfony 6.4, use {@see self::calculateTimeForTokens} instead
```

Comment:
is the deprecation really required?
if yes, this should be accompanied with a runtime deprecation triggered using symfony/deprecation-contracts